modules/dxp/apps/search-experiences/search-experiences-service/src/main/java/com/liferay/search/experiences/internal/ml/embedding/text/TextEmbeddingRetrieverImpl.java

```diff
@@ -42,10 +41,7 @@
 /**
  * @author Petteri Karttunen
  */
-@Component(
-	enabled = false,
-	service = {EmbeddingProviderInformation.class, TextEmbeddingRetriever.class}
-)
+@Component(enabled = false, service = TextEmbeddingRetriever.class)
 public class TextEmbeddingRetrieverImpl implements TextEmbeddingRetriever {
```

Comment:
Hey @MrJohn1911, Could you please add the criteria that you used to remove the extra service in the commit message? Thx